src/Pages/Account/Account.scss

```
@@ -0,0 +1,56 @@
+.myAccount {
+  width: 100%;
+  height: 100%;
+  .myAccountTop {
+    background: url("https://www.logitech.com/content/dam/logitech/my-account/hero-authenticated.png")
```

```suggestion
.myAccount {
  width: 100%;
  height: 100%;
  
  .myAccountTop {
    background: url("https://www.logitech.com/content/dam/logitech/my-account/hero-authenticated.png")
```
- div 태그에 가로 100% 반드시 필요한 상황인가요?
- 새로운 셀렉터 시작할 때 한줄씩 띄워주세요!

---

수정 완료!